docs/marketplace/tutorials/create-a-rag-application.md

```diff
@@ -48,45 +75,60 @@ To begin, navigate to the design section and click on _Applications_ in the side
 First, search for and select the **API Documentation Aggregator**. This will install the [_API Gateway_](/runtime_suite/api-gateway/10_overview.md) and the [_Swagger Aggregator_](/runtime_suite/swagger-aggregator/10_overview.md), along with several endpoints to expose the Swagger UI for viewing the APIs exposed by the configured services and retrieving the OpenAPI compliant schema.
 
 :::note
-The [API Documentation Aggregator](/runtime_suite_applications/api-documentation-aggregator/10_overview.md) is not required for the chatbot to function, but it is recommended for using the _AI RAG Template_ APIs to generate the embeddings. You can also refer to the instructions [in this tutorial](/console/tutorials/configure-marketplace-components/api-portal.mdx).
+The [API Documentation Aggregator](/runtime_suite_applications/api-documentation-aggregator/10_overview.md) is not required for the chatbot to function, but it is recommended for using the _RAG Chatbot API_ APIs to generate the embeddings.
+You can also refer to the instructions [in this tutorial](/console/tutorials/configure-marketplace-components/api-portal.mdx).
 :::
 
-Next, proceed to set up the **AI RAG Chat**. It will prompt you to create the _API Gateway_ (you can use the previously created one with the available listener), the _AI RAG Template_, and the _AI RAG Template Chat_, along with several endpoints to expose the service APIs for sending questions and generating embeddings.
+Next, proceed to set up the application. When searching in the marketplace, search for **AI RAG Chat**. You can find two different results:
+
+- the _AI RAG Chat_, which is the application where the _RAG Chatbot API_ is provided in form of a [plugin](/plugins/mia-platform-plugins.md), maintained and supported by the Mia-Platform team
+- the _AI RAG Chat Template_, which is the application where the _RAG Chatbot API_ is provided in form of a [template](/templates/mia-platform-templates.md) that will create a repository in your Git provider; while this template represents all the features of the plugin and it is fully functional, you can modify it by adding or removing features according to your needs
 
+<!-- TODO: Update this picture with the AI RAG Chat without beta and the AI RAG Chat Template -->
 ![AI RAG Chat in Marketplace](../img/ai-rag-chat-in-marketplace.png)
```

Comment:
The image here must be updated with a new one showing both the applications from the marketplace